spring-amqp/src/main/java/org/springframework/amqp/support/converter/AbstractJsonMessageConverter.java

```diff
@@ -21,13 +21,18 @@
 
 /**
  * Abstract JSON message converter.
+ *
+ * @deprecated - use the {@link org.springframework.amqp.support.converter.AbstractJackson2MessageConverter}.
```

Comment:
`@deprecated` must be in the end of JavaDoc tag list:
```
<module name="AtclauseOrder">
		<property name="target" value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF"/>
   		<property name="tagOrder" value="@param, @author, @since, @see, @version, @serial, @deprecated"/>
</module>
```